src/providers/lambda.ts

```diff
@@ -140,9 +141,48 @@ export class LambdaRunnerProvider extends BaseProvider implements IRunnerProvide
    * Available build arguments that can be set in the image builder:
    * * `BASE_IMAGE` sets the `FROM` line. This should be similar to public.ecr.aws/lambda/nodejs:14.
    * * `EXTRA_PACKAGES` can be used to install additional packages.
+   *
+   * @deprecated Use `imageBuilder()` instead.
    */
   public static readonly LINUX_ARM64_DOCKERFILE_PATH = path.join(__dirname, '..', '..', 'assets', 'docker-images', 'lambda', 'linux-arm64');
 
+  /**
+   * Create new image builder that builds Lambda specific runner images using Amazon Linux 2.
+   *
+   * Included components:
+   *  * `RunnerImageComponent.requiredPackages()`
+   *  * `RunnerImageComponent.runnerUser()`
+   *  * `RunnerImageComponent.git()`
+   *  * `RunnerImageComponent.githubCli()`
+   *  * `RunnerImageComponent.awsCli()`
+   *  * `RunnerImageComponent.githubRunner()`
+   *  * `RunnerImageComponent.lambdaEntrypoint()`
+   *
+   *  Base Docker image: `public.ecr.aws/lambda/nodejs:14-x86_64` or `public.ecr.aws/lambda/nodejs:14-arm64`
+   */
+  public static imageBuilder(scope: Construct, id: string, props?: RunnerImageBuilderProps) {
+    let baseDockerImage = 'public.ecr.aws/lambda/nodejs:14-x86_64';
+    if (props?.architecture === Architecture.ARM64) {
+      baseDockerImage = 'public.ecr.aws/lambda/nodejs:14-arm64';
```

Comment:
need to be able to customize the base image

Reply:
You should still be able to override with `baseDockerImage` as `...props` comes after this.